schema/GenericCamera.yaml

```diff
@@ -102,4 +103,20 @@ allOf:
       filterNumber:
         default: 2
         type: number
-
+- if:
+    properties:
+      camera:
+        const: Canon
+  then:
+    properties:
+      id:
+        default: 0
+        type: number
+      binValue:
+        default: 1
+        type: number
+      ImageType:
+        default: RAW
+        type: string
+        enum:
+          - RAW
```

Comment:
If you can't read the ROI from the camera, you may want to add it as a configuration parameter.